app/components/admin/edition_images/uploaded_images_component.rb

```diff
@@ -4,4 +4,37 @@ class Admin::EditionImages::UploadedImagesComponent < ViewComponent::Base
   def initialize(edition:)
     @edition = edition
```

Comment:
Also thinking we can simplify some logic duplication and improve readability with something like this

```
  def initialize(edition:)
    @edition = edition
    if has_lead_image?
      @lead_image = @edition.images.shift
    end
    @document_images = @edition.images
  end

  def has_lead_image?
    @edition.first_image_pulled_out? && @edition.images.any?
  end

  def lead_image
    image_to_object @lead_image, 0
  end
  
```